src/main/scala/org/ergoplatform/local/TransactionGenerator.scala

```diff
@@ -81,6 +80,47 @@ class TransactionGenerator(viewHolder: ActorRef,
 
     case SuccessfulTransaction(_) => self ! Attempt
   }
+
+  private def genTransaction(wallet: ErgoWallet): Future[Try[ErgoTransaction]] = {
```

Comment:
The idea of transaction generator is to simulate user activity. Thus `genTransaction` should generate transactions, similar to regular users. I doubt regular users will create transactions, that send Ergo, assets and issue new assets at the same time. Let's rewrite it in such a way, that in most cases it generate regular transaction that send ergo, in few cases it generate transactions that send some tokens, in rare cases it generate transactions that issue a new token.